accession-commons-core/pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>accession-commons-core</artifactId>
-    <version>0.6.3</version>
+    <version>0.6.4</version>
     <packaging>jar</packaging>
 
     <parent>
```

Comment:
version was not changed in the parent section for all 5 modules, please change it